src/components/th/th-styles.jsx

```diff
@@ -16,13 +16,22 @@ export default theme => {
       fontFamily: typography.primary.fontFamily,
       fontWeight: typography.fontWeightSemiBold,
 
-      minWidth: 20,
-      paddingBottom: 4,
-
       [mixins.media('md')]: {
         minWidth: 40,
         paddingBottom: 10,
       },
+
+      '&.flexBasis': {
```

Comment:
What do you think about converting these to standalone classes instead? Then they will get generated names from jss.